webpack.common.config.js

```diff
@@ -90,7 +90,7 @@ module.exports = Merge.smart({
             EntitlementSupportPage: './lms/djangoapps/support/static/support/jsx/entitlements/index.jsx',
             PasswordResetConfirmation: './lms/static/js/student_account/components/PasswordResetConfirmation.jsx',
             StudentAccountDeletion: './lms/static/js/student_account/components/StudentAccountDeletion.jsx',
-            StudentAccountDeletionInitializer: './lms/static/js/student_account/StudentAccountDeletionInitializer.js',
+            EdlyStudentAccountDeletionInitializer: './themes/st-lutherx/lms/static/js/student_account/StudentAccountDeletionInitializer.js',
```

Comment:
did you test this change, this file almost create problems for every project. 
see https://github.com/colaraz/edx-platform/pull/40

Reply:
yes, I have tested on local (with old code and new code), and everything works as expected.

Reply:
In case of colaraz there was not theme named `st-lutherx` that was the reason it was throwing error but in case lumsx we have a theme named `st-lutherx` and it has the correct path.